Dockerfile

```diff
@@ -1,9 +1,19 @@
-FROM golang:1.11-alpine
+FROM golang:1.11-alpine as build
 
 RUN apk add --no-cache tzdata
 
 WORKDIR /go/src/github.com/vshn/k8up
 COPY . .
 RUN go install -v ./...
 
-ENTRYPOINT [ "operator" ]
+# runtime image
+FROM docker.io/alpine:3
```

Comment:
Why not `FROM scratch`?

Reply:
The 6MB from the alpine image won't hurt and the overhead of copying things around and hacking them into scratch is not worth it IMO.

Reply:
It somehow "hurts" because it adds unnecessary additional stuff which could increase the attack surface and adds a "base" image maintenance burden. I'm in favor of using `FROM scratch` whenever possible.

Reply:
Will change it to scratch 👍  

@Kidswiss do you know where tzdata should be located (as expected by k8up)?

Reply:
@Kidswiss and I looked at this together and it is too complicated to get it into scratch.